test/views/quote_view_test.exs

```diff
@@ -32,4 +33,12 @@ defmodule HonGraffitiPhoenix.QuoteViewTest do
 
     assert rendered_quote == %{quote: QuoteView.quote_json(quote)}
   end
+
+  test "views return the quote id" do
```

Comment:
In general I agree with the notion that you should add a test for bugged cases when bugs are discovered, but I agree that this is redundant: by adding the line you did to`test "quote_json"`, that test covers this case already. IMO should be removed but I'll leave that up to you.